scripts/gen_mapfiles.py

```diff
@@ -0,0 +1,47 @@
+"""
+    scripts/gen_mapfiles.py
+    ~~~~~~~~~~~~~~~~~~~~~~~
+
+    Regenerate mapping files.
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from importlib import import_module
+from pathlib import Path
+import re
+import sys
+
+top_src_dir = Path(__file__).parent.parent
+pygments_package = top_src_dir / 'pygments'
+sys.path.insert(0, pygments_package.parent.resolve())
+
+from pygments.util import docstring_headline
+
+for key in ['lexers', 'formatters']:
```

Comment:
I'd prefer to move everything here into a function and then do the usual `if __name__ == '__main__':` check, any reason why you're doing this all of this directly?